tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -227,3 +369,140 @@ def test_non_ocs_taint_and_tolerations(self):
                 resource_count=count * replica_count,
             ), "New OSDs failed to reach running state"
             check_ceph_health_after_add_capacity(ceph_rebalance_timeout=2500)
+
+    @skipif_ocs_version("<=4.15")
+    def test_custom_taint_node_operations(self, nodes):
+        """
+        Test runs the following steps
+        1. Add toleration in storagecluster CR and odf-operator subscription.
+        2. Verify toleration applied in ODF subscription is reflecting
+           for other subscriptions, Ceph and nooba components or not(Check all 7 subscriptions).
+        3. Apply custom taint to all nodes.
+        4. Verify the pods in all nodes are running as per taints applied.
+        5. Reboot one of the nodes and check toleration on all odf pods on that node.
+        7.Replace one of the nodes and check all odf pods on that node are running.
+        """
+
+        logger.info("Taint all nodes with custom taint")
+        ocs_nodes = get_worker_nodes()
+        taint_nodes(nodes=ocs_nodes, taint_label="xyz=true:NoSchedule")
+
+        resource_name = constants.DEFAULT_CLUSTERNAME
+        if config.DEPLOYMENT["external_mode"]:
+            resource_name = constants.DEFAULT_CLUSTERNAME_EXTERNAL_MODE
+
+        logger.info("Add tolerations to storagecluster")
```

Comment:
the steps looks almost similar to the existing non ocs taints test, what has changed?